sapi/cli/php_cli_server.c

```diff
@@ -49,6 +50,10 @@
 #include <dlfcn.h>
 #endif
 
+#ifdef __linux__
```

Comment:
you may be able to reuse `HAVE_PRCTL`

Reply:
It's part of fpm and I didn't feel like copying it over was worth it.

Reply:
I meant more making it into configure.ac but that s fine, just detail.

Reply:
furthermore it s used also for naming zend maps, it might be better to detect once and for "everybody".

Reply:
Ok, I can move it to the central configure.ac :slightly_smiling_face: But `PR_SET_PDEATHSIG` is still only available for Linux so I'll need to keep that for the function definition and call (or create a new configure macro)

Reply:
I didn't realize `prctl` wasn't available on FreeBSD at all, so I was able to dodge the `defined(__linux__)` check after all. I moved the m4 stuff to build/php.m4 and called it from configure.ac. I'm not awfully familiar with the build system so let me know if that's the corret approach.